File: test-system-frontend/src/app/_model/member.ts
import {Result} from './result';
import {Test} from './test';
export class Member{
  id: bigint;
  name: string;
  surname: string;
  email: string;
  password: string;
  role: string;
  token: string;
  results: Result[] = [];
  tests: Test[] = [];
}

File: test-system-backend/src/main/java/ua/knu/montag/spring/model/Member.java
package ua.knu.montag.spring.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.NaturalId;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;

@Entity(name="Member")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Member {
    @Id
    @GeneratedValue
    private Long id;
    private String name;
    private String surname;
    @NaturalId
    private String email;
    private String password;
    private String role;
    @OneToMany(mappedBy = "member", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Result> results = new ArrayList<>();
    @OneToMany(mappedBy = "member", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Test> tests = new ArrayList<>();

}
